src/server/migrations/5_create_user_belong_project_table.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE userBelongProject (
```

Comment:
A pretty common pattern for many-to-many table naming is `firsttable__secondtable`, so here it would be `users__project`.